src/main/components/form/validator.ts

```diff
@@ -13,7 +13,7 @@ export const isFieldFilledIn: Validator = value => {
 export const isRespondentNameValid: Validator = value => {
   if (!value || (value as string).trim().length === 0) {
     return 'required';
-  } else if (!/(=?^.{5,100}$)/.test(value as string)) {
+  } else if (!/(=?^.{1,100}$)/.test(value as string)) {
```

Comment:
This change doesn't seem to be covered by tests. Validator should allow minimum two characters

Reply:
Respondent name can be 1 character, see comments https://tools.hmcts.net/jira/browse/RET-1827